hack/ds-kustomize/ds.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app.kubernetes.io/name: ds
     app.kubernetes.io/part-of: forgerock
 spec:
-  # The number of DS servers in the topology. 
+  # The number of DS servers in the topology.
```

Comment:
Maybe we should say something like `The number of DS replicas`?  In multi-cluster this won't be accurate as this will be half of the topology